test/e2e/skopeo.go

```diff
@@ -13,17 +13,17 @@ import (
 
 const (
 	insecure              = "--insecure-policy=true"
-	skopeo                = "skopeo"
+	Skopeo                = "Skopeo"
 	debug                 = "--debug"
 	skipTLS               = "--dest-tls-verify=false"
 	skipCreds             = "--dest-no-creds=true"
 	destCreds             = "--dest-creds="
 	v2format              = "--format=v2s2"
-	skopeoImage           = "quay.io/olmtest/skopeo:0.1.40"
+	skopeoImage           = "quay.io/olmtest/Skopeo:0.1.40"
```

Comment:
do we want to change the name of the skopeo image tag? 

In general, I can see the motivation of exporting all this code out of the `e2e` package so that the `utils` package you wrote can access it. I think its better if we simply moved all this existing skopeo/registry code out of the `e2e` package into `utils` and export from `utils` into `e2e` only whats necessary. 

The e2e package is way too big already :)

Reply:
This was a refactoring error, got that fixed. Looks like buildah does a great job of pushing bundles on its own, so we won't need this.